crates/indexer/src/http/store_config.rs

```diff
@@ -104,37 +113,84 @@ pub async fn process(client: &Client, config_key: Pubkey, uri_str: String) -> Re
         .context("failed to insert store config json")?;
 
     if let Some(creators) = json.creators {
+        client
+            .db()
+            .run({
+                let addr = addr.clone();
+                move |db| {
+                    let remove_creators = store_creators::table
+                        .filter(store_creators::store_config_address.eq(&addr))
+                        .select(store_creators::creator_address)
+                        .get_results::<String>(db)
```

Comment:
Should this not also be inside the transaction?

Reply:
moved to within the transaction.